bundles/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/model/Panel.java

```diff
@@ -188,50 +167,41 @@ public void handleEvent(Item item, DSCAlarmBindingConfig config, EventPublisher
                 if (config.getDSCAlarmItemType() != null) {
                     switch (config.getDSCAlarmItemType()) {
                         case PANEL_CONNECTION:
-                            state = panelProperties.getSystemConnection();
                             publisher.postUpdate(item.getName(), new DecimalType(state));
                             break;
                         case PANEL_MESSAGE:
                             if (apiMessage != null) {
                                 str = apiMessage.getAPIDescription();
-                                panelProperties.setSystemMessage(str);
                             }
                             publisher.postUpdate(item.getName(), new StringType(str));
                             break;
 
                         case PANEL_SYSTEM_ERROR:
-                            panelProperties.setSystemError(1);
-                            panelProperties.setSystemErrorCode(0);
                             int systemErrorCode = 0;
 
                             if (apiMessage != null) {
                                 systemErrorCode = Integer.parseInt(apiMessage.getAPIData());
-                                panelProperties.setSystemErrorCode(systemErrorCode);
-                                panelProperties.setSystemErrorDescription(apiMessage.getError());
-                                str = String.format("%03d", panelProperties.getSystemErrorCode()) + ": " + panelProperties.getSystemErrorDescription();
+                                str = String.format("%03d", systemErrorCode) + ": " + apiMessage.getError();
```

Comment:
Minor: it could be more efficient to String.format the entire string, not just part and then use string concatenation.